manifold-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,196 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
```

Comment:
Let's call this dnsLabelPrefix to make it clear to the user submitting parameter values what this is and the rules that are needed for this to be succesful. See #966